README.md

```diff
@@ -65,6 +65,38 @@ function setAction(Action $action) {
 }
 ```
 
+## Comparison
+
+Php-enum are not singletons and it has an important implication when it comes to comparing values. The following code will _not_ work as you would expect it to work:
+
+```php
+// returns false
+if ($action === Action::VIEW()) {
+  // ...
+}
+```
```

Comment:
In my experience many people don't read paragraphs and copy/paste the first code block they find.

I'd rather start this section with "how to compare enums" (i.e. the code examples you have written below), and then explain that comparing enum instances with `===` doesn't work. No need to write more than a sentence or two TBH. No need to link to the issue either, nor use the word "workaround": this isn't really a bug, this is a minor detail that hasn't prevented this library to become the most widespread enum library in PHP.